airbyte-integrations/connectors/source-gitlab/acceptance-test-config.yml

```diff
@@ -0,0 +1,26 @@
+connector_image: airbyte/source-gitlab:dev
+tests:
+  spec:
+    - spec_path: "source_gitlab/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+      validate_output_from_all_streams: yes
+#    We cannot use these tests for testing Incremental, since for Gitlab the State is saved for each Project separately,
```

Comment:
we know the projects we are working with so we should be able to hardcode the path no?

Reply:
@sherifnada this is true, but how SAT will know that each value should be used for comparison against the value in record?

Reply:
WDYM? as in which field in the record? isn't that just the cursor field?

Reply:
if there are multiple records like :
```json
{
   "project1": {"state": value1},
   "project1": {"state": value2},
}
```
the SAT needs to know how to compare a particular record[cursor_field] with these list of states

Reply:
but if there is only one account then we should be ok

Reply:
@yevhenii-ldv I think with one account we can hardcode paths and enable incremental test

Reply:
@keu @sherifnada  I tried to do as you suggest and hardcode paths for test, but it didn't work.
At the moment, we do not support the functionality for transferring several parameters to `cursor_paths`.
The `get_state_value` method inside Class `JsonSchemaHelper` does not support this.
```
    def get_state_value(self, state, cursor_path):
        format_ = self.get_format_for_key_path(path=cursor_path)
        value = state[cursor_path[-1]]
        return self.parse_value(value, format_)
```
I had to tweak the code of this method a little locally, and the tests really all started and passed (I will attach a screenshot of this).
However, at this point in time, before changing the SAT, I had to comment out the Incremental tests.

![Screenshot from 2021-07-02 15-54-54](https://user-images.githubusercontent.com/34103125/124280972-ef62de00-db51-11eb-8d11-491c00cc5f24.png)

Reply:
@yevhenii-ldv good enough, thanks for going the extra mile here. 

Can you create an issue to add this functionality to SAT, comment this block out from SAT config and link to the issue in the comment? (also create a child issue to fix the Gitlab SAT config once the parent issue is resolved)